charts/datahub/subcharts/datahub-mae-consumer/values.yaml

```diff
@@ -207,7 +207,7 @@ global:
       #  secretKey: <secret-key>
 
     ## Enables always emitting a MCL even when no changes are detected. Used for Time Based Lineage when no changes occur.
-    alwaysEmitChangeLog: true
+    alwaysEmitChangeLog: false
```

Comment:
Why are changes being made at the sub chart level? Shouldn't it just be in the top-level values.yaml?